### PR TITLE
Updates to Zootropolis video page

### PR DESCRIPTION
**Pull request by lps88**
## What does this change?
- use brand colour in appropriate places
- refactor the video template a little
- add link to zootropolis article page from video
- fix page URLs for VB and renault campaigns
### Before:

image
### After:

image

@guardian/labs-beta

### CODE REVIEW

**Comment by Calanthe** [comment]
Its seems ok, maybe ask @kelvin-chappell for the scala feedback. 👍

File: common/app/common/commercial/hosted/HostedGalleryPage.scala [hunk 1]
```diff
@@ -22,6 +22,7 @@ case class HostedGalleryPage(
 
   val pageTitle: String = s"Advertiser content hosted by the Guardian: $title - gallery"
   val twitterText = shareText.getOrElse(if(standfirst.length < 136) standfirst else title) + " #ad"
+  val imageUrl = images.head.url
```
Comment (JonNorman) [inline common/app/common/commercial/hosted/HostedGalleryPage.scala]: Is `images` always going to be populated? You can use `.headOption` to defensively cope with an empty sequence.

Comment (lps88) [inline common/app/common/commercial/hosted/HostedGalleryPage.scala]: Hmm I hadn't thought about that. Though a gallery without any images isn't really a gallery is it?! 😸

Comment (JonNorman) [inline common/app/common/commercial/hosted/HostedGalleryPage.scala]: Ha, it is definitely a minimalist one! I just recall a bug where we were getting `.tail` on an empty list and it broke the page so always worth checking :)

File: common/app/common/commercial/hosted/hardcoded/ZootropolisHostedPages.scala [hunk 2]
```diff
@@ -84,6 +84,10 @@ object ZootropolisHostedPages {
     customData
   )
 
+
+  private lazy val videoPage = if (Switches.hostedArticle.isSwitchedOn) videoPageWithoutNextPage
```
Comment (kelvin-chappell) [inline common/app/common/commercial/hosted/hardcoded/ZootropolisHostedPages.scala]: Might want to decouple article switch from video switch, or rename it if campaign can only launch in one go.

File: common/app/common/commercial/hosted/hardcoded/RenaultHostedPages.scala [hunk 3]
```diff
@@ -28,7 +29,7 @@ object RenaultHostedPages {
     val videoTitle = "Designing the car of the future"
     HostedVideoPage(
       campaign,
-      pageUrl = "https://www.theguardian.com/commercial/advertiser-content/renault-car-of-the-future/design-competition-teaser",
+      pageUrl = s"$host/commercial/advertiser-content/renault-car-of-the-future/design-competition-teaser",
```
Comment (kelvin-chappell) [inline common/app/common/commercial/hosted/hardcoded/RenaultHostedPages.scala]: Does `host` include protocol?

Comment (lps88) [inline common/app/common/commercial/hosted/hardcoded/RenaultHostedPages.scala]: it looks like it does (http, should it be https?)

Comment (kelvin-chappell) [inline common/app/common/commercial/hosted/hardcoded/RenaultHostedPages.scala]: It should be https, but may be dangerous to change it sitewide at the moment.  So I think this is fine.

**Comment by kelvin-chappell** [comment]
👍

**Pull request merged**